linkstone-annotations/src/main/java/net/glowstone/linkstone/annotations/LGenerate.java

```diff
@@ -0,0 +1,15 @@
+package net.glowstone.linkstone.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation that tells the linkstone compiler to generate
+ * a getter and setter that accesses the annotated field.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface LGenerate {
+}
```

Comment:
Can `@LGetter` and `@LSetter` do this instead, when applied to fields rather than to methods? Otherwise it's too confusingly different from Lombok.

Reply:
The `@LGetter` and `@LSetter` annotation **are** different from Lombok. That's why explicitly prefixed them with an 'L'.

How would your usage of the annotation look like?
Can you send the example from the [wiki](https://github.com/GlowstoneMC/Linkstone/wiki/Working-with-Fields) with your annotations style?